mmv1/templates/terraform/encoders/resource_compute_region_network_firewall_policy_with_rules.go.erb

```diff
@@ -0,0 +1,3 @@
+delete(obj, "rules") // Rules are not supported in the create API 
```

Comment:
Could you provide a link to where the Rules is not supported by the create API? I've been reviewing the PR and noticed that the `insert` for adding rules contains `rules` as part of the request body. Maybe I'm not referencing the correct resource?

https://cloud.google.com/compute/docs/reference/rest/v1/regionNetworkFirewallPolicies/insert

Reply:
thanks for clarification!